media-gfx/openscad/openscad-2015.03_p2-r3.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
EAPI bump requires revision bump at first

Reply:
Done for both versions.